code/modules/vehicles/armored/vehicle_collision.dm

```diff
@@ -29,16 +29,19 @@
 
 /turf/closed/wall/vehicle_collision(obj/vehicle/sealed/armored/veh, facing, mob/pilot, ram_damage = veh.ram_damage)
 	take_damage(ram_damage, BRUTE, MELEE, TRUE, REVERSE_DIR(facing), 0)
+	if(istype(veh, /obj/vehicle/sealed/armored/multitile/mrap)) //TODO flags: only for testing, otherwise it will conflict with other prs
+		veh.take_damage(ram_damage * 0.4, BRUTE, MELEE, TRUE, veh.dir, 0, pilot)
```

Comment:
https://tenor.com/de/view/the-office-i-will-kill-whisper-i-will-cut-you-gif-20241258

Reply:
I can't set the flags without it conflicting with other PR's I am TM'ing, review faster stinky